src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -219,6 +213,10 @@ func LibraryArchivedBookPath(archivePath, book string) string {
 }
 
 
+func (c *Client) BookIAM(book *librarypb.Book) *iam.Handle {
+    return iam.InternalNewHandle(c.Connection(), book.Name)
+}
```

Comment:
This is self-inconsistent. If you are removing AddTag/AddLabel, your replacement accessor function should match them semantically; they should be on the topic of tagging/labeling instead of IAM.